lib/matplotlib/widgets.py

```diff
@@ -1938,10 +1939,28 @@ class SpanSelector(_SelectorWidget):
 
     span_stays : bool, default: False
         If True, the span stays visible after the mouse is released.
+        Deprecated, use interactive instead.
+
+    interactive : bool, default: False
+        Whether to draw a set of handles that allow interaction with the
+        widget after it is drawn.
```

Comment:
To me the primary difference is whether the selector is still visible after the first selection.

When `interactive=True` I get a selector that persists, but I can still modify later. Coming from that context I would expect `interactive=False` to still show my selected region, but I cannot modify it anymore. This indicates that `interactive` is a misleading name.

When starting from scratch, I'd call this `persistent`, but given that we have `span_stays` keeping that is ok as well.